README.md

```diff
@@ -303,6 +304,35 @@ fn main() { run().unwrap() }
 &nbsp;&NewLine;&nbsp;&NewLine;&nbsp;&NewLine;
 
 
+<a id="itertools"></a>
+### `itertools = "0.6.0"` &emsp; [📖][d-itertools]
+
+Itertools provides the extra iterator adaptors, functions and
+macros.
```

Comment:
The word "the" here is extraneous.

I think we can say tell a more interesting story in this description. Something like

"The Rust standard `Iterator` (hyperlinked) type provides a powerful abstraction for operating over sequences of values,
and is used pervasively throughout Rust. There are though a number of common operations one might want to perform on sequences that are not provided by the standard library, and that's where `itertools` comes in. If you've ever found yourself missing ... etc."

stdx actually used to contain itertools, and [here's the text it used to use](https://github.com/brson/stdx/commit/eb0d625f5d1ed87936e74a04063bcc8368882fc4#diff-04c6e90faac2675aa89e2176d2eec7d8L114), which is suitably colorful.